### PR TITLE
Initial classes added

### PR DESCRIPTION
**Pull request by Alibaba2023**
# Changes in this milestone 💹☄

- [X] Class Person added
- [X] Class Student added
- [X] Class Teacher added

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  def initialize(age, name = 'unkown', parent_permession: true)
+    @id = rand
+    @name = name
+    @parent_permession = parent_permession
+    @age = age
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permession
```
Comment (ichala) [inline person.rb]: - [ ] for the can_use_services? kindly use the of_age? method for testing 🙏 

 def can_use_services? 
    of_age?() || @parent_permission
  end

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permession
```
Comment (ichala) [inline person.rb]: - [ ] please note that the `:id should only be able to be retrieved so only the Getter is defined` , kindly update your code by adding `attr_reader :id` 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Created an individual `.rb` file for each class (`Person`, `Student`, and `Teacher` )✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @Alibaba2023 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.
- All classes created successfully.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**